src/nimble.nim

```diff
@@ -615,15 +615,23 @@ proc listPaths(options: Options) =
       if kind != pcDir or not path.startsWith(options.getPkgsDir / name):
         continue
 
-      let
-        nimbleFile = path / name.addFileExt("nimble")
-        hasSpec = nimbleFile.existsFile
-
-      if hasSpec:
+      var nimbleFile = path / name.addFileExt("nimble")
```

Comment:
Why don't you just use the `findNimbleFile` proc here?

Reply:
Err. Because we still need to know whether it is a link? Specifically, the `nimbleLinkTargetPath` value.